ethereum/core/src/main/java/org/hyperledger/besu/ethereum/vm/GasCalculator.java

```diff
@@ -438,4 +438,9 @@ default Gas modExpGasCost(final Bytes input) {
   default long getMaxRefundQuotient() {
     return 2;
   };
+
+  // what would be the gas for a PMT with hash of all non-zeros
+  default Gas getMaximumPmtCost() {
```

Comment:
Do we really need the default implementation?

Reply:
removed